### PR TITLE
Add tui-visualization for resources and tasks

### LINKED ISSUE #23: Monitoring REST + CLI
A very simple CLI should be able to connect to the current scheduling "daemon" and dump current state (resource and allocations, etc.). This ticket only requires a minimal output alike a proof of concept.

We can later extend this as we add more features. This CLI could be a great tool for integration testing

### PR DESCRIPTION
**Pull request by neithanmo**
This makes it possible to see the current status of the scheduler. showing resources state and task state
closes #23

### CODE REVIEW

File: monitor/src/main.rs [hunk 1]
```diff
@@ -84,23 +101,30 @@ fn main() -> Result<(), String> {
...
     } else if let Some(matches) = matches.subcommand_matches("abort") {
         let id: &str = matches
             .value_of("client")
             .ok_or_else(|| "No client_id arg".to_string())?;
         let id = id.parse::<u64>().map_err(|e| e.to_string())?;
-        abort(address, id)
+        abort(&address, id).map_err(|e| e.into())
     } else {
-        Err("Niether monitor nor abort sub-command".to_string())
+        Err("Nither monitor nor abort sub-command".to_string().into())
```
Comment (jleni) [inline monitor/src/main.rs:128]: Minor typo here

**Review by jleni — CHANGES_REQUESTED** [review]
Should we add some brief instructions for the CLI in README.md?

**Comment by jleni** [comment]
Didn't try it.. but tui-rs looks great so I can already imagine this will looks great too :)

**Comment by leongb** [comment]
Yes indeed was thinking of the same: maybe add some documentation to the readme? Because I don't know how to get out of the monitor process? Feels like VIM haha

**Pull request merged**